src/styles.tsx

```diff
@@ -0,0 +1,9 @@
+import styled from "styled-components";
+
+const GreyTitle = styled.h1`
+  color: #8b9293;
```

Comment:
for colors use `theme`

Reply:
I can't use the appTheme hook here, because its not inside a react component

Reply:
you don't need a hook here, the `theme` inject in `styled-component` inside `DeskproAppProvider` [here](https://github.com/DeskproApps/app-sdk/blob/master/src/ui/context/DeskproAppProvider.tsx#L79)

Reply:
try to use like this
```javascript
const GreyTitle = styled.h1`
  color: ${({ theme }) => theme.color.grey60};
`
```